app: Enable boot logging (#319)
nrfconnect/ncs-serial-modem: doc/app/at_trace.rst

The Zephyr application log backend (``AT#XLOG``) and the modem trace backend (``AT#XTRACE``) share a single UART.
They are mutually exclusive: enabling one while the other is active returns an error.
During boot, the UART outputs logs from B0, MCUboot, and the application.

> After application initialization, the UART is suspended when neither backend is active and resumed when either backend is enabled.

Reviewer: ```suggestion
After initialization, the UART is suspended when no backend is active and resumed when at least one backend is enabled.
```
Please check the meaning.
